### PR TITLE
Repositioning and styling adjustments

### LINKED ISSUE #73: Connection between report notes and call log unclear
Test subjects do not understand the connection between the notes entered by themselves during the "report" step, and the call log in the "prepare" step. They do not understand where the notes they enter end up.

Hence, they also do not consider the fact that some information (such as a changed address) does not belong in the notes.

_This was observed (noted as signal 13) during the december 2016 user tests._

### LINKED ISSUE #93: Phone number difficult to find
Test subjects have a hard time finding phone numbers, especially in the "prepare" step (where phone numbers are not visible). They do not understand the need to proceed to the "call" step before the number appears.

This happens especially after a call has been disrupted and the subject returns to the prepare step.

_This was observed (noted as signal 33) during the december 2016 user tests._

### PR DESCRIPTION
**Pull request by jensborje**
This PR moves some elements around from bottom to top and vice versa.

Relevant for #92 
Fixes #93 
Fixes #73

### CODE REVIEW

**Comment by jensborje** [comment]
Styling for ReportPane.

skarmavbild 2016-12-17 kl 16 26 08

skarmavbild 2016-12-17 kl 16 26 01

skarmavbild 2016-12-17 kl 16 26 40

File: locale/misc/sv.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 callLog:
+    note: "Ingen anteckning lämnad"
```
Comment (richardolsson) [inline locale/misc/sv.yaml]: Is there any particular reason you decided to not mention the caller as suggested in #73?

File: src/components/misc/callLog/CallLogItem.jsx [hunk 2]
```diff
@@ -39,15 +39,10 @@ export default class CallLogItem extends React.Component {
...
-                <div ref="notes"
-                    className="CallLogItem-notes">
-                    {   call.get('notes') }
-                </div>
-            );
-        }
+        // TODO: Add class when no note and style these.
+        let notes = (call.get('notes').length > 0)
+            ? call.get('notes')
+            : (<Msg id="misc.callLog.note"/>);
```
Comment (richardolsson) [inline src/components/misc/callLog/CallLogItem.jsx]: Parentheses should not be necessary here.

File: src/components/misc/callLog/CallLogItem.jsx [hunk 3]
```diff
@@ -39,15 +39,10 @@ export default class CallLogItem extends React.Component {
...
-        let notes = null;
-        if (call.get('notes').length > 0) {
-            notes = (
-                <div ref="notes"
-                    className="CallLogItem-notes">
-                    {   call.get('notes') }
-                </div>
-            );
-        }
+        // TODO: Add class when no note and style these.
```
Comment (richardolsson) [inline src/components/misc/callLog/CallLogItem.jsx]: This TODO makes sense. Can we do that before we merge?

**Pull request merged**